csharp/src/Ice/OutputStream.cs

```diff
@@ -856,7 +856,7 @@ public void WriteTaggedEnum(int tag, int? v)
                 if (elementSize > 1)
                 {
                     // This size is redundant and optimized out by the encoding when elementSize is 1.
-                    WriteSize(v.Length == 0 ? 1 : (v.Length * elementSize) + (v.Length > 254 ? 5 : 1));
+                    WriteSize(v.Length == 0 ? 1 : v.Length * elementSize + GetSizeLength(v.Length));
```

Comment:
I think is nice to keep the parenthesis for clarity

Reply:
Adding them back.